.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: ${{ github.event.inputs.project_name }}
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+# Controls when the action will run.
+on:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      project_name:
+        description: 'The github repository name'
+      tag_name:
+        description: 'The tag name on the project repo to be released'
+      project_version:
+        description: 'Version number to be released e.g 10.0.3'
+      fancy:
+        description: 'If fancy versioning is required set it "True"'
+        default: 'False'
+      fancy_version_no:
+        description: 'If fancy versioning is True, then it should be set e.g 1'
+        default: '1'
+      microsoft_email:
+        description: 'Email to be written on changelogs'
+        default: 'gindibay@microsoft.com'
+      name:
+        description: 'Name to be written on changelogs'
+        default: 'Gurkan Indibay'
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    # The type of runner that the job will run on
```

Comment:
```suggestion
```

`runs-on: ubuntu-latest` is really clear and it does not need any comments IMO

Reply:
Fixed